boards/common/arduino-atmega/include/arduino_pinmap.h

```diff
@@ -225,6 +225,33 @@ extern "C" {
 #define ARDUINO_A14             ADC_LINE(14)
 #define ARDUINO_A15             ADC_LINE(15)
 #endif
+
+// A majority of the pins are NOT PCINTs, SO BE WARNED (i.e. you cannot use them as receive pins)
+// Only pins available for RECEIVE (TRANSMIT can be on any pin):
+// (I've deliberately left out pin mapping to the Hardware USARTs - seems senseless to me)
+// Pins: 10, 11, 12, 13,  50, 51, 52, 53,  62, 63, 64, 65, 66, 67, 68, 69
+
+#define digitalPinToPCICR(p)    ( (((p) >= 10) && ((p) <= 13)) || \
+                                  (((p) >= 50) && ((p) <= 53)) || \
+                                  (((p) >= 62) && ((p) <= 69)) ? (&PCICR) : ((uint8_t *)0) )
```

Comment:
Yes, this comes from https://github.com/arduino/ArduinoCore-avr/tree/master/variants
I've moved it to `sys/arduino/include/pins_arduino.h` (chose the `pins_arduino.h` filename because that's the way it's called in ArduinoCore-avr).

Added the docs.. Is there a way to avoid repeating the documentation? (The macros are defined several times, since they depend on the board).